api/src/main/java/edu/cornell/mannlib/vitro/webapp/dynapi/RESTEndpoint.java

```diff
@@ -0,0 +1,63 @@
+package edu.cornell.mannlib.vitro.webapp.dynapi;
+
+import java.io.IOException;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+
+import edu.cornell.mannlib.vitro.webapp.controller.VitroHttpServlet;
+
+@WebServlet(name = "RESTEndpoint", urlPatterns = { "/api/rest/*" })
+public class RESTEndpoint extends VitroHttpServlet  {
+
+	private static final long serialVersionUID = 1L;
+ 	private static final Log log = LogFactory.getLog(RESTEndpoint.class);
+	private ResourcePool resourcePool = ResourcePool.getInstance();
+ 	
+	
+	@Override
+  public void service(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
```

Comment:
As a note, we may want to have consistent formatting. Not sure what style VIVO follows more.